deploy/rebuild-app.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Migrate backend
+cd /home/ubuntu/blackout/blackout
+python3 manage.py migrate
+
+# Build frontend
+cd ../frontend
+npm run build
```

Comment:
Okey, you build, but where you do npm run preview?

Reply:
There is no need to run preview, because i don't need to start a server with npm, i just need to build the project into dist directory, and then it will be deployed by ngrok.

There is docs that say i dont need this:
https://vitejs.dev/guide/static-deploy.html

Reply:
Ok